tests/repositories/snapshots/cargo_ci/.github/workflows/cargo-rust-ci-docker-application.yml.snap

```diff
@@ -2,7 +2,7 @@
 source: tests/common/mod.rs
 expression: content
 ---
-name: cargo_library-docker-app
+name: cargo-rust-ci-docker-app
```

Comment:
Why does it change here to cargo-rust-ci, starting from cargo_library? However, the context is correct.